extension/extension_config.cmake

```diff
@@ -3,7 +3,7 @@
 #add_static_link_extension(fts)
 #add_static_link_extension(httpfs)
 #add_static_link_extension(iceberg)
-#add_static_link_extension(json)
+add_static_link_extension(json)
```

Comment:
Why is this line uncommented in this PR?

Reply:
Commented this line